src/mdtest.c

```diff
@@ -2161,122 +2136,122 @@ mdtest_results_t * mdtest_run(int argc, char **argv, MPI_Comm world_com, FILE *
+            o.nstride *= numTasksOnNode0;
         }
-        VERBOSE(0,5,"Shifting ranks by %d for each phase.", nstride);
+        VERBOSE(0,5,"Shifting ranks by %d for each phase.", o.nstride);
     }
 
-    VERBOSE(3,-1,"main (after display_freespace): testdirpath is '%s'", testdirpath );
+    VERBOSE(3,-1,"main (after display_freespace): o.testdirpath is '%s'", o.testdirpath );
 
     if (rank == 0) {
-        if (random_seed > 0) {
-            VERBOSE(0,-1,"random seed: %d", random_seed);
+        if (o.random_seed > 0) {
+            VERBOSE(0,-1,"random seed: %d", o.random_seed);
         }
     }
 
-    if (gethostname(hostname, MAX_PATHLEN) == -1) {
+    if (gethostname(o.hostname, MAX_PATHLEN) == -1) {
         perror("gethostname");
         MPI_Abort(testComm, 2);
     }
 
     if (last == 0) {
-        first = size;
-        last = size;
+        first = o.size;
+        last = o.size;
     }
 
     /* setup summary table for recording results */
-    summary_table = (mdtest_results_t *) malloc(iterations * sizeof(mdtest_results_t));
-    memset(summary_table, 0, iterations * sizeof(mdtest_results_t));
+    o.summary_table = (mdtest_results_t *) malloc(iterations * sizeof(mdtest_results_t));
+    memset(o.summary_table, 0, iterations * sizeof(mdtest_results_t));
     for(int i=0; i < iterations; i++){
       for(int j=0; j < MDTEST_LAST_NUM; j++){
-        summary_table[i].rate[j] = 0.0;
-        summary_table[i].time[j] = 0.0;
+        o.summary_table[i].rate[j] = 0.0;
+        o.summary_table[i].time[j] = 0.0;
       }
     }
 
-    if (summary_table == NULL) {
+    if (o.summary_table == NULL) {
         FAIL("out of memory");
     }
 
-    if (unique_dir_per_task) {
-        sprintf(base_tree_name, "mdtest_tree.%d", rank);
+    if (o.unique_dir_per_task) {
+        sprintf(o.base_tree_name, "mdtest_tree.%d", rank);
     } else {
-        sprintf(base_tree_name, "mdtest_tree");
+        sprintf(o.base_tree_name, "mdtest_tree");
     }
 
     /* default use shared directory */
-    strcpy(mk_name, "mdtest.shared.");
-    strcpy(stat_name, "mdtest.shared.");
-    strcpy(read_name, "mdtest.shared.");
-    strcpy(rm_name, "mdtest.shared.");
+    strcpy(o.mk_name, "mdtest.shared.");
+    strcpy(o.stat_name, "mdtest.shared.");
+    strcpy(o.read_name, "mdtest.shared.");
+    strcpy(o.rm_name, "mdtest.shared.");
 
     MPI_Comm_group(testComm, &worldgroup);
 
     /* Run the tests */
-    for (i = first; i <= last && i <= size; i += stride) {
+    for (i = first; i <= last && i <= o.size; i += stride) {
         range.last = i - 1;
         MPI_Group_range_incl(worldgroup, 1, (void *)&range, &testgroup);
         MPI_Comm_create(testComm, testgroup, &testComm);
         if (rank == 0) {
-            uint64_t items_all = i * items;
-            if(num_dirs_in_tree_calc){
-              items_all *= num_dirs_in_tree_calc;
+            uint64_t items_all = i * o.items;
+            if(o.num_dirs_in_tree_calc){
```

Comment:
The normal style in the rest of the code is "if (" and ") {} (with a space in between).  Minor whitespace cleanups like this could also be done in this patch.